.github/workflows/release_pypi.yml

```diff
@@ -0,0 +1,104 @@
+name: PyPI Release
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

Comment:
Please change this to Python 3.9 (assuming it works). I like to keep all my automation current and Python 3.6 is nearly end of lifed.